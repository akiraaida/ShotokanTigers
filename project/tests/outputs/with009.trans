10 Ashe Ace             00000 00000.00 S 
03 Ashe Ace             12348 02000.00 TV
01 Ashe Ace             12348 00000.10   
01 Ashe Ace             12348 00005.00   
01 Ashe Ace             12348 00000.10   
02 Ashe Ace             12348 01000.00   
02 John Doe             12345 01000.00   
01 Ashe Ace             12348 00000.10   
01 Ashe Ace             12348 00005.00   
01 Ashe Ace             12348 00000.10   
00                      00000 00000.00   
