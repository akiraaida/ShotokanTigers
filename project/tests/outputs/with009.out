Please enter your session type:
Please enter a login name:
Please enter a command user:
Please enter the account number:
Please enter recipient to be paid:
Please enter amount to pay:
Success, 2000.00 has been paid to TV.
Available Balance (12348):
8009.90
Daily Payment Limit to TV Remaining (12348):
0.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12348):
8004.80
Daily Withdrawals Remaining (12348):
495.00
Please enter the account number:
Enter account number to transfer to:
Enter amount:
Success, 1000.00 has been transferred to 12345.
Available Balance (12348):
7004.70
Available Balance (12345):
1110.00
Daily Transfer Limit Remaining (12348):
0.00
Please enter the account number:
Please enter amount to withdraw:
Success, 5.00 has been withdrawn.
Available Balance (12348):
6999.60
Daily Withdrawals Remaining (12348):
490.00
